src/main/cpp/MpvAvPlaybackPipeline.cpp

```diff
@@ -0,0 +1,267 @@
+#include "MpvAvPlaybackPipeline.h"
```

Comment:
Ideally, having separate error codes for the MPV player would be the best approach. In the MpvAVPlayback.h  I am using the code errors of the MPV client header but in the MpvPlaybackPipeline.h I am using ffmpeg ERROR_FFMPEG_UNKNOWN code for any error returned by the MPV Playback to be dispatched to the Java side. This might be added in the next sprint if not done today.